docs/data/joy/components/aspect-ratio/aspect-ratio.md

```diff
@@ -7,58 +7,71 @@ title: React Aspect Ratio component
 
 <p class="description">The aspect ratio component shapes the content with the specified ratio.</p>
 
-The default aspect ratio implementation combines `height: 0px` with percentage padding-bottom to create the proper aspect ratio for the content.
-(It will be replaced by the [CSS aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) in the future once it has sufficient browser support).
+## Introduction
 
-- The content to fit the area of the aspect ratio component must be its first direct child.
-- It's designed to be composed with other components, e.g. [`Card`](/joy-ui/react-card/).
+`AspectRatio` is a wrapper component that allows you to rapidly control its content aspect ratio.
+Its default implementation combines `height: 0px` with percentage `padding-bottom` to properly accomodate the content.
 
-## Basic usage
+:::info
+**Note:** A [native CSS `aspect-radio`property](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) already exists but we're not using it yet due to limited browser support.
+Once that increases significantly, we'll switch over to it.
+:::
+
+## Component
+
+After [installation](/joy-ui/getting-started/installation/), you can start building with this component using the following basic elements:
```

Comment:
404 https://mui.com/joy-ui/getting-started/installation/

x all the other instances.

Reply:
Yup, as of now it only exists on this branch!

Reply:
Ahh ok, I assumed that it was the same problem as in https://github.com/mui/material-ui/pull/33288/files#diff-92cf14b7b3e7031bde9aa0ea77a8176a918d9cda9235258575a9ea6ada9af4c0R23.